scripts/run_stubtest.py

```diff
@@ -8,15 +8,26 @@
 
 import pandas as pd
 
+pd_version = getattr(pd, "__version__", "")
+
 # fail early if pandas is not installed
-if not getattr(pd, "__version__", ""):
+if not pd_version:
     # fail on the CI, soft fail during local development
     warnings.warn("You need to install the development version of pandas")
     if pd.compat.is_ci_environment():
         sys.exit(1)
     else:
         sys.exit(0)
 
+# GH 48260
+if "dev" not in pd_version:
+    warnings.warn(
+        f"stubtest may fail as {pd_version} is not a dev version. "
+        f"Please install a pandas dev version or see <https://github.com/pandas-dev"
```

Comment:
1. Could you remove the `<` and `>`?
2. Could you link to `https://pandas.pydata.org/pandas-docs/stable/development/contributing_codebase.html#validating-type-hints` instead?

Reply:
Thanks for the review :)

Resolved the two issues you mentioned